src/PresentationalComponents/RuleDetails/RuleDetails.js

```diff
@@ -0,0 +1,185 @@
+    };
+
+    render () {
+        const { children, className, ruleFetchStatus, rule } = this.props;
+        const { kbaDetails } = this.state;
+        return (
+            <>
+                { ruleFetchStatus === 'pending' && (<Loading/>) }
+                { ruleFetchStatus === 'fulfilled' && (
+                    <Grid gutter='md' className={ className }>
+                        <GridItem md={ 8 } sm={ 12 }>
+                            <Grid>
+                                <GridItem className='pf-u-pb-md'>
+                                    {
+                                        typeof rule.summary === 'string' &&
+                                        Boolean(rule.summary) &&
+                                        <ReactMarkdown source={ rule.summary } escapeHtml={ false }/>
+                                    }
+                                </GridItem>
+                                { kbaDetails.view_uri && (
+                                    <GridItem className='pf-u-pb-md'>
+                                        <a href={ kbaDetails.view_uri }>
+                                            Knowledgebase Article
+                                        </a>
+                                    </GridItem>
+                                ) }
+                                { rule.tags && (
+                                    <GridItem>
+                                        Find Other Rules Related To:<br />
+                                        { rule.tags.replace(/ /g, ', ') }
+                                    </GridItem>
+                                ) }
+                            </Grid>
+                        </GridItem>
+                        <GridItem md={ 4 } sm={ 12 }>
+                            <Grid gutter='sm'>
+                                { children && (
+                                    <GridItem>
+                                        { children }
+                                    </GridItem>
+                                ) }
+                                <GridItem className='pf-u-pb-md' sm={ 8 } md={ 12 }>
+                                    <div>Total Risk</div>
+                                    <div className='pf-u-display-inline-flex pf-m-align-center pf-u-pb-sm pf-u-pt-sm'>
+                                        <div className='pf-u-display-inline-flex'>
+                                            <Battery
+                                                label=''
+                                                severity={ rule.total_risk }
+                                            />
+                                        </div>
+                                        <div className='pf-u-display-inline-flex'>
+                                            <span className={ `label pf-u-pl-sm ins-sev-clr-${rule.total_risk}` }>
+                                                &nbsp;{ AppConstants.TOTAL_RISK_LABEL[rule.total_risk] }
```

Comment:
So when this is undefined, the label goes away, maybe add an `|| 'Undefined' after `AppConstants.TOTAL_RISK_LABEL[rule.total_risk]` ?